### PR TITLE
Installing and Verifying Python 2.7

### PR DESCRIPTION
**Pull request by annegentle**

### CODE REVIEW

File: labs/02-python-01-home-lab-python/5.md [hunk 1]
```diff
@@ -1,48 +1,46 @@
...
+Note: for Python 3.x, check the “add to path” checkbox (see sample below) during installation.
 
-      2.  Python 3.x: Check the “add to path” checkbox (see sample below) during installation.
-
-Note that on Windows, you should take the extra steps to put the **python** command in your system’s execution path. The Python installer, by default, does not add the directories with the Python commands in them to the PATH variable. You should override that default at installation, adding Python to your PATH. For Python 2.7, you have to first select the option to use custom settings for the installation, and then look for the option to add the commands to the path, as shown on the left side of the figure. The Python 3.6 process puts the checkbox on a screen that is part of each system’s installation process, as shown on the right side of the figure.
+On Windows, take the extra steps to put the **python** command in your system’s execution path. The Python installer, by default, does not add the directories with the Python commands in them to the PATH variable. You should override that default at installation, adding Python to your PATH.
```
Comment (annegentle) [inline labs/02-python-01-home-lab-python/5.md]: I removed the pull-out numbered list item for Python 3 as this section seems entirely about 2.7. I also removed the redundant paragraph in line 35.

File: labs/02-python-01-home-lab-python/5.md [hunk 2]
```diff
@@ -1,48 +1,46 @@
...
+The Python installation process follows a typical Windows application installation process. If you already have a recent Python 2 version installed and verified, you can skip this step. If not, assuming you want to install the most recent Python 2.7 release, use these steps:
 
-1.  Navigate to [www.python.org](http://www.python.org)
+1.  Navigate to [www.python.org](http://www.python.org).
 
-2.  Click the [Downloads menu item](https://www.python.org/downloads/) at the top of the page
+2.  Click the [Downloads menu item](https://www.python.org/downloads/) at the top of the page. The page lists the most recent 2.7 and 3.6 release download links. 
 
-3.  The page should list the most recent 2.7 and 3.x release download at the top of the page, for the OS of the system from which you are web browsing. Click that link, or follow the navigation to list [all the Windows versions of Python](https://www.python.org/downloads/windows/) for Windows.
+3.  Click the link for the 2.7 version, or go to the list of [all the Windows versions of Python](https://www.python.org/downloads/windows/).
```
Comment (annegentle) [inline labs/02-python-01-home-lab-python/5.md]: I tested these instructions on Windows 10 and modified slightly for the version I saw.

Comment (WendellOdom) [reply labs/02-python-01-home-lab-python/5.md]: I agree with this change.

File: labs/02-python-01-home-lab-python/5.md (hunk 1 above)
Comment (WendellOdom) [reply labs/02-python-01-home-lab-python/5.md]: Anne, I'd like to reject both of these changes (but not aware of the mechanism to do so.) While the page is primarily about Python 2.7, I believe both points are important in context to a learner. If documentation, I'd choose to take them out. But the learner will be thinking about both versions, and I think the content as listed here is worth keeping.

Comment (annegentle) [inline labs/02-python-01-home-lab-python/5.md]: Okay, I'll revise again.

Comment (annegentle) [reply labs/02-python-01-home-lab-python/5.md]: Okay, the mechanism is to do an inline comment (not in the "Conversation" tab but in the "Files Changed" tab I believe. Then you should have a button labeled "Start a Review" and you can then click the "Review Changes" green drop-down in the upper right corner and click the "Request Changes" radio button. 

I'll address in a revision to keep the context for the learner.

**Pull request merged**